File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.go
import (
    "strings"
    "strconv"
)
func solution(s string) string {
    var min, max int
    nums := strings.Split(s, " ")
    min, _ = strconv.Atoi(nums[0])
    max, _ = strconv.Atoi(nums[0])
    
    
    
    for _, str := range strings.Split(s, " ") {
        num, err := strconv.Atoi(str)
        if err != nil {
            panic("panic")
        }
        if num < min {
            min = num
        }
        if num > max {
            max = num
        }
        
    }
    
    return strconv.Itoa(min) + " " + strconv.Itoa(max)
}

File: Python3/프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    list_string = []
    for num in s.split() :
        list_string.append(int(num))
    
    min_num, max_num = list_string[0], list_string[0]
    for num in list_string :
        if min_num > num : min_num = num
        if max_num < num : max_num = num
    
    
    return str(min_num) + " " + str(max_num)